src/components/common/tags/Tags.vue

```diff
@@ -1,23 +1,33 @@
 <template>
   <div
-    v-for="tag in tags"
-    :key="tag"
-    class="flex"  
+    class="flex flex-wrap items-center py-3 text-sm text-white font-medium"
   >
-    <Tag :tag="tag"/>
-  </div> 
+    <div 
+      v-for="tag in tags" 
+      :key="tag" 
+      class="flex">
+        <Tag 
```

Comment:
move v-for directly on Tag element, does not make sense to add an extra div wrapper here